doc/conf.py

```diff
@@ -170,15 +167,6 @@
 #modindex_common_prefix = []
 
 nitpicky = True
-nitpick_ignore = [ ('cpp:typeOrConcept', 'std')
-                 , ('cpp:typeOrConcept', 'std::shared_ptr')
-                 , ('cpp:typeOrConcept', 'std::ostream')
-                 , ('cpp:typeOrConcept', 'std::string')
-                 , ('cpp:typeOrConcept', 'std::string_view')
-                 , ('cpp:typeOrConcept', 'std::tuple')
-                 , ('cpp:typeOrConcept', 'V') # template arguments which should be matched but aren't.
-                 , ('cpp:typeOrConcept', 'Args')
-                 ]
```

Comment:
Note to reviewers: With the upgrade to sphinx 1.7.5, these issues seem to be fixed such that we can parse normally, so @SolidWallOfCode suggested we remove the exceptions while leaving the `nitpicky` alone.